chain/types/event.go

```diff
@@ -0,0 +1,24 @@
+package types
+
+import (
+	"github.com/filecoin-project/go-address"
+)
+
+type Event struct {
+	// The ID of the actor that emitted this event.
+	Emitter address.Address
+
+	// Key values making up this event.
+	Entries []EventEntry
+}
+
+type EventEntry struct {
+	// A bitmap conveying metadata or hints about this entry.
+	Flags uint8
+
+	// The key of this event entry
+	Key []byte
+
+	// Any DAG-CBOR encodeable type.
+	Value []byte
```

Comment:
Are we sure we want to use a raw bytes type here, or `cbor.Deferred` (IIRC)?

Reply:
Good point. Though cbor.Deferred is easy enough to derive from a raw byte slice. We don't use cbor.Deferred in any other types. Perhaps the closest semantics are message params which uses []byte